### DISCUSSION TITLE
Can we plot triangle in CIE 1931 with custom points

### DISCUSSION THREAD

**Discussion by nitishn23** [colour-science/colour#1093] [Q&A] [answered]
Hello ,

I would like to plot a triangle in CIE 1931 colour space as below image.
![gamut](https://user-images.githubusercontent.com/78595294/212036291-249faa5a-11f6-46a2-bc11-402b525b8bf9.jpg)
Cutsom triangle coordinates: 
CIE 1931 x  CIE 1931 y
0.690	   0.310
0.170	  0.720
0.150	  0.030

**Comment by thomasmansencal** [comment, marked as answer]
Hi @nitishn23,

Yes this is easy:

```python
import colour

RGB_COLOURSPACE_CUSTOM = colour.RGB_Colourspace(
    "Custom",
    [[0.690, 0.310], [0.170, 0.720], [0.150, 0.030]],
    colour.CCS_ILLUMINANTS["CIE 1931 2 Degree Standard Observer"]["D65"],
)

colour.plotting.plot_RGB_colourspaces_in_chromaticity_diagram_CIE1931(
    RGB_COLOURSPACE_CUSTOM, plot_kwargs={"color": "y"}
)
```

![image](https://user-images.githubusercontent.com/99779/212147666-582c8948-72da-4fd1-9e3b-50f449e8a839.png)

**Reply by nitishn23** [reply]
Hi @KelSolaar Thanks this works well for me.
![image](https://user-images.githubusercontent.com/78595294/212276105-e8243be8-b3d9-478a-9382-7b01963de0ad.png)
 Any idea how to get these highlighted x,y coordinates of colour selected on the plot, my purpose is I want to pass these coordinates to different function. Allow user to select the colour from the region highlted and get the coordinates

**Reply by thomasmansencal** [reply]
Matplotlib’s Event Handling should give you click coordinates: https://matplotlib.org/stable/users/explain/event_handling.html

**Reply by nitishn23** [reply]
Thanks @KelSolaar i could work it out with this, I was trying to convert XYZ to RGB values using this XYZ_to_RGB. RGB=XYZ_to_RGB(
            XYZ,
            illuminant_XYZ,
            illuminant_RGB,
            matrix_XYZ_to_RGB,
            )

My output is like this  RGB=[  85.56690369   26.05828243  155.7562224 ], How can i get these individual values of these array when i try accessing RGB[0], RGB[1],RGB[2] it gives me a wrong value.

**Reply by thomasmansencal** [reply]
Are you sure your output is 1D? Would you mind printing `RGB.shape` please?

**Reply by nitishn23** [reply]
Ohh now i get the values correct, earlier i was scaling them to 255 and printing the individual array values that creates a problem.
Thank for the quick response
